dev-tools/validation/test_adr_review_status.py

```
@@ -0,0 +1,493 @@
+        for status in ("Accepted", "Superseded by ADR 0099", "Withdrawn"):
+            with self.subTest(status=status), fixture_root() as fixture:
+                root = Path(fixture)
+                path = root / "design/architecture/decisions/adr-0013-pending.md"
+                path.write_text(
+                    path.read_text(encoding="utf-8").replace(
+                        "Proposed - Pending Human Review", status
+                    ),
+                    encoding="utf-8",
+                )
+                expect_failure(self,
+                    lambda: self.validator.validate(root),
```

Ruff (0.15.21)

[warning] Function definition does not bind loop variable `root`

(B023)